packages/webview_flutter/webview_flutter_android/android/src/test/java/io/flutter/plugins/webviewflutter/WebChromeClientTest.java

```diff
@@ -120,9 +120,13 @@ public void onCreateWindow() {
   public void onPermissionRequest() {
     final PermissionRequest mockRequest = mock(PermissionRequest.class);
     instanceManager.addDartCreatedInstance(mockRequest, 10);
-
     webChromeClient.onPermissionRequest(mockRequest);
-
     verify(mockFlutterApi).onPermissionRequest(eq(webChromeClient), eq(mockRequest), any());
   }
+
+  @Test
+  public void onGeolocationPermissionsHidePrompt() {
+    webChromeClient.onGeolocationPermissionsHidePrompt();
+    verify(mockFlutterApi).onGeolocationPermissionsHidePrompt(eq(webChromeClient), any());
+  }
```

Comment:
What about testing for the show callback?

Reply:
@bparrishMines  Could you help me to write this, I'm not able to write this, thanks

Reply:
This Java test is also don't know how to write. @stuartmorgan

Reply:
The question is the same here: what specifically are you having trouble with?

Reply:
@stuartmorgan For context, I wrote the wrapper part with my generator as mentioned here: https://github.com/flutter/packages/pull/3475#discussion_r1157171020

This was in the old closed PR, so you wouldn't have been able to find that info.

@yk3372 I can take care of the java tests

Reply:
> @stuartmorgan For context, I wrote the wrapper part with my generator as mentioned here: [#3475 (comment)](https://github.com/flutter/packages/pull/3475#discussion_r1157171020)
> 
> This was in the old closed PR, so you wouldn't have been able to find that info.
> 
> @yk3372 I can take care of this java test

Thanks. ^_^